frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -572,6 +656,31 @@ export function ContainerInfo(props: ContainerInfoProps) {
       env[envVar.name] = value;
     });
 
+    function preparePortForwardComponent(containerPort: number) {
+      if (!helpers.isElectron()) {
+        return null;
+      }
+      return !portForward ? (
+        <Button
+          onClick={() => handlePortForward(namespace, resource?.metadata.name, containerPort)}
+        >
+          {t('frequent|Port Forward')}
+        </Button>
```

Comment:
We need to set up some feedback for the user to know this is happening. Maybe just having a spinner after we click the button until we hear back from the call?

Reply:
We do a show a loader using the portForward object